src/core/container.rs

```diff
@@ -140,6 +140,11 @@ impl<'d, I> Container<'d, I> {
             })
     }
 
+    /// Returns ip address of docker container as specified in NetworkSettings.IPAddress
+    pub fn get_ip_address(&self) -> String {
```

Comment:
One more thing: Given that Docker involves quite a few different network layers, I think we need to more specific here about which exact IP address this is.

If I am not mistaken, the IP address returned here will be the one of the `bridge` network for communicating with the docker container from the host machine?

Reply:
That's my understanding. I couldn't find any detailed documentation on it, but I did compare a bridge container to a host container and that value was only populated in the bridge container. So that confirms it isn't just taking the address of the first network it finds at least.